src/zenml/zen_server/routers/triggers_endpoints.py

```diff
@@ -81,54 +80,10 @@ def list_triggers(
     Returns:
         All triggers.
     """
-
```

Comment:
This was a way to interpose the action handler between trigger REST API calls and the database as a sort of admission controller. Now that the actions are detached, there's no immediate need for this, but in the future, when we have a proper event hub implementation, the event hub will basically have to play that same role: intervene in every trigger REST API call and act as an admission controller. In other words the final event hub class should implement `get_trigger`, `create_trigger`, `update_trigger` and `delete_trigger` and basically talk to whatever back-end implements the message hub to keep it updated with these events.